client/js/views/templates/OurOfferings.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function( { model } ) {
+    return `<div> our offerings </div>`
```

Comment:
Keep out the whitespace before and after our offerings and implement padding or margin.